buildSrc/src/main/kotlin/Utilities.kt

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Kotlin has the concept of [Extension functions](https://kotlinlang.org/docs/extensions.html).
Sadly, those cannot be "exposed" from a Gradle plugin, i.e. from plugins defined in the included build `buildSupport`, so these extension functions need to live in `buildSrc`, which is an integral part of the whole build.